src/tests/routes/users/snapshots/crates_io__tests__routes__users__admin__get__admin-found.snap

```diff
@@ -0,0 +1,17 @@
+---
+source: src/tests/routes/users/admin.rs
+expression: response.json()
+---
+{
+  "avatar": null,
+  "email": "foo@example.com",
+  "email_verification_sent": true,
+  "email_verified": true,
+  "id": 1,
+  "is_admin": false,
+  "lock": null,
+  "login": "foo",
+  "name": null,
+  "publish_notifications": true,
+  "url": "https://github.com/foo"
+}
```

Comment:
the rest of the API is using e.g. `{ user: { ... } }`, which allows us to side-load other data from the same request in some cases. we should probably stay consistent with the other API endpoints.